_includes/pbc/all/install-features/202307.0/install-the-shipment-customer-account-management-feature.md

```diff
@@ -0,0 +1,246 @@
+        ];
+    }
+}
+```
+
+## 3) Set up widgets
+
+Register the following plugins to enable widgets:
+
+| PLUGIN                        | SPECIFICATION                                                 | PREREQUISITES | NAMESPACE                                  |
+|-------------------------------|---------------------------------------------------------------|---------------|--------------------------------------------|
+| ShipmentTypeAddressFormWidget | Enables shipment type selection during the checkout address step. | None          | SprykerShop\Yves\ShipmentTypeWidget\Widget |
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\ShipmentTypeWidget\Widget\ShipmentTypeAddressFormWidget;
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @return string[]
+     */
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            ShipmentTypeAddressFormWidget::class,
+        ];
+    }
+}
+```
+
+## 4) Set up FE part
+
+Adjust TWIG templates to display the shipment types:
+
+1. Single shipment.
+
+Add `ShipmentTypeAddressFormWidget` to the `address` view of `CheckoutPage` module.
+
+```twig
+{% raw %}{% widget 'ShipmentTypeAddressFormWidget' args [data.form] with {
+    data: {
+        deliveryContainerClassName: deliveryContainerClassName,
+        billingSameAsShippingContainerClassName: embed.jsAddressClass ~ '__wrapper-billingSameAsShipping',
+    },
+} only %}
+{% endwidget %}{% endraw %}
+```
+
+{% info_block infoBox "Info" %}
+
+`deliveryContainerClassName` - class name of container the delivery form and the address selector.<br>
+`embed.jsAddressClass ~ '__wrapper-billingSameAsShipping'` - class name of container for billing same as shipping checkbox.
+
+{% endinfo_block %}
+
+2. Multi shipment.
+
+{% info_block infoBox "Info" %}
+
+You can skip this step if multi-shipment isn't used on the project.
+
+{% endinfo_block %}
+
+a) Add the same widget as for single shipment.
+
+b) Add `multiple-shipment-toggler` molecule to the `address` view of `CheckoutPage` module.
+
+```twig
+{% raw %}{% include molecule('multiple-shipment-toggler', 'CheckoutPage') with {
+    data: {
+        isMultipleShipmentSelected: isMultipleShipmentSelected,
+    },
+    attributes: {
+        'toggle-targets-class-name': singleDeliveryContainerClassName,
+        'select-class-name': deliverySelectClassName,
+    },
+} only %}{% endraw %}
+```
+
+{% info_block infoBox "Info" %}
+
+`isMultipleShipmentSelected` - flag that indicates if multiple shipment is selected.<br>
+`singleDeliveryContainerClassName` - class name of container the address selector and the `ShipmentTypeAddressFormWidget`.<br>
```

Comment:
In this case, I meant the 'wrapper of address selector`, so we don't need a comma between. <br> tag also needed, since `info_block` doesn't render new line without it.

Reply:
@andrew-maslov, then adjust the phrase so that there're no grammar mistekes. You can't put "the" between "container" and "address" as this is dramatically incorrect.
It should be like "class name of the container address selector," for example.

Regarding `<br>`, the infoblock renders just fine without `<br>` (tested). You just need to have an extra line between sentences.